blog/views.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.urls import reverse_lazy
+from django.views import generic
+from blog.models import Post, Commentary
```

Comment:
Add blank lines between different groups of imports according to PEP-8 in the future